### PR TITLE
bug/recently-added-query-incorrect

### PR DESCRIPTION
**Pull request by jonhindle**
### Context

> Does this issue have a Trello card?
https://trello.com/c/eShastSg/1030-bug-recently-added-query-using-incorrect-url

> If this is an issue, do we have steps to reproduce?
View the recently added home page component or page.

### Intent

> What changes are introduced by this PR that correspond to the above card?
query url updated

> Would this PR benefit from screenshots?
n/a

### Considerations

> Is there any additional information that would help when reviewing this PR?
n/a

> Are there any steps required when merging/deploying this PR?
n/a

### Checklist

- [ ] This PR contains **only** changes related to the above card
- [ ] Tests have been added/updated to cover the change
- [ ] Documentation has been updated where appropriate
- [ ] Tested in Development

### CODE REVIEW

File: server/repositories/cmsQueries/__tests__/recentlyAddedContentQuery.spec.js [hunk 1]
```diff
@@ -12,7 +12,7 @@ describe('Recently Added Content page query', () => {
...
...upal_internal__target_id%5D%5Bcondition%5D%5Bvalue%5D%5B2%5D=moj_radio_item&filter%5Btype.meta.drupal_internal__target_id%5D%5Bcondition%5D%5Bvalue%5D%5B3%5D=moj_pdf_item&filter%5Btype.meta.drupal_internal__target_id%5D%5Bcondition%5D%5Boperator%5D=IN&filter%5Bcreated%5D%5Bvalue%5D=1651737416&filter%5Bcreated%5D%5Boperator%5D=%3E%3D&include=field_moj_thumbnail_image&sort=-published_at%2Ccreated&fields%5Bnode--page%5D=drupal_internal__nid%2Ctitle%2Cfield_moj_thumbnail_image%2Cfield_moj_description%2Cfield_moj_series%2Cpath%2Ctype.meta.drupal_internal__target_id&fields%5Bnode--moj_video_item%5D=drupal_internal__nid%2Ctitle%2Cfield_moj_thumbnail_image%2Cfield_moj_description%2Cfield_moj_series%2Cpath%2Ctype.meta.drupal_internal__target_id&fields%5Bnode--moj_radio_item%5D=drupal_internal__nid%2Ctitle%2Cfield_moj_thumbnail_image%2Cfield_moj_description%2Cfield_moj_series%2Cpath%2Ctype.meta.drupal_internal__target_id&fields%5Bnode--moj_pdf_item%5D=drupal_internal__nid%2Ctitle%2Cfield_moj_thumbnail_image%2Cfield_moj_description%2Cfield_moj_series%2Cpath%2Ctype.meta.drupal_internal__target_id&fields%5Bfile--file%5D=drupal_internal__fid%2Cid%2Cimage_style_uri&page[offset]=0&page[limit]=40`,
```
Comment (leonk) [inline server/repositories/cmsQueries/__tests__/recentlyAddedContentQuery.spec.js:15]: You don’t need the content type filter in there, this is already taken care of

**Review by BarryGeeMOJ — APPROVED** [review]
lgtm

**Review by BarryGeeMOJ — DISMISSED** [review]
lgtm

File: server/repositories/cmsQueries/__tests__/recentlyAddedContentQuery.spec.js [hunk 2]
```diff
@@ -12,7 +12,7 @@ describe('Recently Added Content page query', () => {
...
+        `/jsonapi/prison/${ESTABLISHMENTNAME}/recently-added?filter%5Bcreated%5D%5Bvalue%5D=1651737416&filter%5Bcreated%5D%5Boperator%5D=%3E%3D&include=field_moj_thumbnail_image&sort=-published_at%2Ccreated&fields%5Bnode--page%5D=drupal_internal__nid%2Ctitle%2Cfield_moj_thumbnail_image%2Cfield_moj_description%2Cfield_moj_series%2Cpath%2Ctype.meta.drupal_internal__target_id&fields%5Bnode--moj_video_item%5D=drupal_internal__nid%2Ctitle%2Cfield_moj_thumbnail_image%2Cfield_moj_description%2Cfield_moj_series%2Cpath%2Ctype.meta.drupal_internal__target_id&fields%5Bnode--moj_radio_item%5D=drupal_internal__nid%2Ctitle%2Cfield_moj_thumbnail_image%2Cfield_moj_description%2Cfield_moj_series%2Cpath%2Ctype.meta.drupal_internal__target_id&fields%5Bnode--moj_pdf_item%5D=drupal_internal__nid%2Ctitle%2Cfield_moj_thumbnail_image%2Cfield_moj_description%2Cfield_moj_series%2Cpath%2Ctype.meta.drupal_internal__target_id&fields%5Bfile--file%5D=drupal_internal__fid%2Cid%2Cimage_style_uri&page[offset]=0&page[limit]=40`,
```
Comment (leonk) [inline server/repositories/cmsQueries/__tests__/recentlyAddedContentQuery.spec.js:15]: Another thing... I don't think we need the created filter either (that's just for the full page).

Comment (jonhindle) [reply server/repositories/cmsQueries/__tests__/recentlyAddedContentQuery.spec.js:15]: It's the same query atm. I'm guessing them being the same mean we'll get a cache pay off somewhere along the line 😬

**Review by leonk — APPROVED** [review]
LGTM

**Pull request merged**